packages/foreman/nodejs-gud/nodejs-gud.spec

```diff
@@ -1,16 +1,24 @@
+%{?scl:%scl_package nodejs-%{npm_name}}
+%{!?scl:%global pkg_name %{name}}
+
 %global npm_name gud
 
-Name: nodejs-gud
+Name: %{?scl_prefix}nodejs-gud
 Version: 1.0.0
-Release: 1%{?dist}
+Release: 2%{?dist}
 Summary: Create a 'gud nuff' (not cryptographically secure) globally unique id
 License: MIT
 Group: Development/Libraries
 URL: https://github.com/jamiebuilds/global-unique-id#readme
 Source0: https://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
+%if 0%{?scl:1}
+BuildRequires: %{?scl_prefix_nodejs}npm
```

Comment:
Why does this need NPM? This is a plain package and not bundled.

Reply:
This is already in half the packages. So I'd prefer to come back around and make updates as a whole since this has little impact.